application-analytics-test/application-analytics-test-docker/src/test/it/xwiki/analytics/test/ui/config/Config.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ */
+package xwiki.analytics.test.ui.config;
+
+public class Config
+{
+    public static final String DB_CONTAINER_NAME = "farcasut/custom-mysql:latest";
```

Comment:
The test needs to run on CI. I doubt this custom Docker container is published somewhere, but even if it is, we should use an official container.

Reply:
The container is currently publicly hosted on Docker Hub, but do you think it would be a better choice to create the container from scratch? Like using the default container and populating the database using the API provided by TestContainers